src/pages/Items/AddItem/AddItem.tsx

```diff
@@ -33,13 +46,56 @@ const AddItem = () => {
           [name]: true,
         };
       });
+      setInputValues((prev) => ({
+        ...prev,
+        [name]: currentValue,
+      }));
     } else if (currentValue.length < 1) {
       setIsValid((prev) => {
         return {
           ...prev,
           [name]: false,
         };
       });
+      setInputValues((prev) => ({
+        ...prev,
+        [name]: currentValue,
+      }));
+    }
+  };
+
+  const imgFileUpload = async (imgFile: FormData) => {
+    const result = await postUploadImage(imgFile);
+    if (result) {
+      setInputValues((prev) => ({
+        ...prev,
+        images: result.url,
+      }));
+    }
+  };
+
+  const uploadPostMutation = useMutation({
```

Comment:
보통 이 부분을 구조분해할당을 이용해서 바로 가져옵니다. 이렇게 작성하면 아래에서 대략 어떤 코드가 이어질지 예상할 수 있다는 장점이 있습니다!

```suggestion
  const { mutate, isPending } = useMutation({
```